### DISCUSSION TITLE
Mulighet til å endre posisjonen til en tavle

### DISCUSSION THREAD

**Discussion by PeterRydberg** [entur/tavla#309] [Ideas]
Basisposisjonen (geografisk område) til en tavle blir nå satt idet man oppretter tavla, men det er ikke mulig å endre på dette i ettertid. Derfor hadde det vært fint om man kunne velge et nytt område via admin-panelet (f.eks. ved bruk av samme søkefelt som på forsiden), i tilfelle brukeren ikke er fornøyd med sin opprinnelige posisjon eller vil benytte samme tavle til å vise stoppesteder innenfor en ny range på et helt annet sted.

**Comment by lindtvedtsebastian** [comment]
This discussion is linked to the old (deprecated) version of Tavla and will therefore be closed.
